### PR TITLE
Appt Cards, Profile pic upload button

### PR DESCRIPTION
**Pull request by shashanklal01**
updated-card-date, added fields to cards, included upload button for profile picture

### CODE REVIEW

**Comment by ishchhabra** [comment]
Since this pull request contains the profile pic upload task as well. I was wondering whether I should convert pull request to draft while I work on the backend of the same and link them as updating profile pic button without having the functionality to actually use it would leave us with no profile pic at all?

**Comment by ishchhabra** [comment]
Hello,
I just updated the backend files containing backward compatibility a little. So, until we review this, the website will still run smoothly. The only difference will be that all the images stored into MySQL right now are useless, and all currently stored profile pictures will be not shown. You will, instead, need to edit the profile again and enter the photo again.

What we are basically doing is just creating a file in s3 (profile/{id}/image) which contains a string. Now, after frontend changes, this string will be the base64 encoded image which user inputs with the FileUpload button. However, right now, this will just store a link. So, this achieves us backward compatibility while we review the stuff.

File: lambdas/get_supporter_settings.py [hunk 1]
```diff
@@ -184,4 +184,19 @@ def get_supporter_settings(event, context):
     return {
         'body': block,
         'statusCode': 200
-    }
\ No newline at end of file
+    }
+
+def get_profile_picture(student_id):
```
Comment (victoriacaruso) [inline lambdas/get_supporter_settings.py:189]: student_id should be supporter_id

**Comment by ishchhabra** [comment]
Merged this branch into backend directly since it was already approved by victoria

**Pull request merged**